torchvision/transforms/functional.py

```diff
@@ -572,7 +572,7 @@ def resized_crop(
             :class:`torchvision.transforms.InterpolationMode`.
             Default is ``InterpolationMode.BILINEAR``. If input is Tensor, only ``InterpolationMode.NEAREST``,
             ``InterpolationMode.BILINEAR`` and ``InterpolationMode.BICUBIC`` are supported.
-            For backward compatibility integer values (e.g. ``PIL.Image.NEAREST``) are still acceptable.
+            For backward compatibility integer values (e.g. ``PIL.Image(.Resampling).NEAREST``) are still acceptable.
```

Comment:
I'm ok with this style in the docstring. Any objections?

Reply:
`()` is fine, although as a nit, I think backets `[]` are more common (typically in `man` pages)

Reply:
@kylematoba Could you address this in all docstrings?

Reply:
Sure, so replace instances of `()` with `[]` in my changes thusfar, and a second PR that just removes all of the lines about `For backward compatibility integer values`?

Reply:
> Sure, so replace instances of `()` with `[]` in my changes thusfar,

Yes.


> and a second PR that just removes all of the lines about `For backward compatibility integer values`?

No. The docstring need to be changed indicating that passing `int`'s is actually deprecated, e.g.

https://github.com/pytorch/vision/blob/62740807c18e68bb0acd85895dca527f9a655bd5/torchvision/transforms/functional.py#L1029-L1031

Furthermore, we need to change the warning 

https://github.com/pytorch/vision/blob/62740807c18e68bb0acd85895dca527f9a655bd5/torchvision/transforms/functional.py#L415-L419

to include the concrete deprecation date, e.g.

https://github.com/pytorch/vision/blob/62740807c18e68bb0acd85895dca527f9a655bd5/torchvision/transforms/functional.py#L1042-L1045

For this we will be deprecating in `0.13` and remove in `0.15`.